marklogic/assets/dashboards/overview.json

```diff
@@ -0,0 +1,781 @@
+            "id": 8732386596109986,
+            "layout": {
+                "height": 8,
+                "width": 15,
+                "x": 148,
+                "y": 22
+            }
+        },
+        {
+            "definition": {
+                "background_color": "white",
+                "content": "Datadog's integration for MarkLogic gives you the visibility you need to identify performance issues and tune your deployments more effectively.\n\nThis dashboard shows an overview of your cluster health, and provides more information about forests and databases.\n\nClone this template dashboard to make changes and add your own graph widgets.",
+                "font_size": "14",
+                "show_tick": false,
+                "text_align": "left",
+                "tick_edge": "left",
+                "tick_pos": "50%",
+                "type": "note"
+            },
+            "id": 2191273978937394,
+            "layout": {
+                "height": 27,
+                "width": 22,
+                "x": 0,
+                "y": 14
+            }
+        },
+        {
+            "definition": {
+                "legend_size": "0",
+                "requests": [
+                    {
+                        "display_type": "line",
+                        "q": "sum:marklogic.hosts.total_rate{$cluster} by {host}",
+                        "style": {
+                            "line_type": "solid",
+                            "line_width": "normal",
+                            "palette": "dog_classic"
+                        }
+                    }
+                ],
+                "show_legend": false,
+                "time": {},
+                "title": "Sum of the processing rate factors over $cluster",
+                "title_align": "left",
+                "title_size": "16",
+                "type": "timeseries",
+                "yaxis": {
+                    "include_zero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "id": 5983605072600952,
+            "layout": {
+                "height": 15,
+                "width": 47,
+                "x": 24,
+                "y": 65
+            }
+        },
+        {
+            "definition": {
+                "background_color": "vivid_blue",
+                "content": "Database summary",
+                "font_size": "14",
+                "show_tick": false,
+                "text_align": "center",
+                "tick_edge": "left",
+                "tick_pos": "50%",
+                "type": "note"
+            },
+            "id": 3172903413969796,
+            "layout": {
+                "height": 5,
+                "width": 38,
+                "x": 128,
+                "y": 0
+            }
+        },
+        {
+            "definition": {
+                "background_color": "blue",
```

Comment:
Can we make this a category (rather than a subcategory) and add a "#" before "Requests"?